cpp/src/arrow/dataset/file_parquet.h

```diff
@@ -175,7 +175,7 @@ class ARROW_DS_EXPORT ParquetFileFragment : public FileFragment {
   Result<std::shared_ptr<Fragment>> Subset(std::vector<int> row_group_ids);
 
   static std::optional<compute::Expression> EvaluateStatisticsAsExpression(
-      const Field& field, const parquet::Statistics& statistics);
+      const Field& field, const FieldRef& field_ref, const parquet::Statistics& statistics);
```

Comment:
This is public API? So I can add a variant with the original signature that creates a FieldRef from the Field?